File: sort.cpp
#include <string>
#include <algorithm>
#include <iostream>

std::string sort(double width, double height, double length, double mass) {
    double volume = width * height * length;
    bool is_bulky = volume >= 1000000 || std::max({width, height, length}) >= 150;
    bool is_heavy = mass >= 20;
    
    if (is_bulky && is_heavy) {
        return "REJECTED";
    } else if (is_bulky || is_heavy) {
        return "SPECIAL";
    } else {
        return "STANDARD";
    }
}

int main() {
    double width, height, length, mass;
    std::cout << "Width (cm): ";
    std::cin >> width;
    std::cout << "Height (cm): ";
    std::cin >> height;
    std::cout << "Length (cm): ";
    std::cin >> length;
    std::cout << "Mass (kg): ";
    std::cin >> mass;
    std::cout << sort(width, height, length, mass) << std::endl;
    return 0;
}

File: Sort.java
import java.util.Scanner;

public class Sort {
    public static String sort(double width, double height, double length, double mass) {
        double volume = width * height * length;
        boolean isBulky = volume >= 1000000 || Math.max(Math.max(width, height), length) >= 150;
        boolean isHeavy = mass >= 20;
        
        if (isBulky && isHeavy) {
            return "REJECTED";
        } else if (isBulky || isHeavy) {
            return "SPECIAL";
        } else {
            return "STANDARD";
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Width (cm): ");
        double width = scanner.nextDouble();
        System.out.print("Height (cm): ");
        double height = scanner.nextDouble();
        System.out.print("Length (cm): ");
        double length = scanner.nextDouble();
        System.out.print("Mass (kg): ");
        double mass = scanner.nextDouble();
        System.out.println(sort(width, height, length, mass));
        scanner.close();
    }
}
